src/web/client/extension.ts

```diff
@@ -89,7 +92,11 @@ export function activate(context: vscode.ExtensionContext): void {
                         );
                     }
                 }
-
+                const geo = queryParamsMap.get('geo')?.toUpperCase();
+                // Authenticated scenario. Pass the geo to OneDSLogger for data boundary
+                if(geo !== undefined && geo !== null && geo !== ''){
```

Comment:
instead of these checks can we replace it by simply checking if geo has a **truthy** value?

Reply:
fixed